internal/pkg/agent/application/monitoring/inputs.go

```diff
@@ -0,0 +1,36 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package monitoring
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/elastic/elastic-agent-libs/monitoring"
+)
+
+func inputsHandler(ns *monitoring.Namespace) func(http.ResponseWriter, *http.Request) error {
+	return func(w http.ResponseWriter, r *http.Request) error {
+		w.Header().Set("Content-Type", "application/json; charset=utf-8")
+
+		data := monitoring.CollectStructSnapshot(
```

Comment:
this should be exposed in filebeat right? why are we adding inputs handler here?

Reply:
I removed this inputs handler.